about.html

```diff
@@ -70,4 +91,33 @@
       </footer>
     </div><!-- container -->        
 </body>
-</html>
\ No newline at end of file
+</html>
+
+
+<!-- 
+  
+  
+If you were to describe semantic HTML to the next cohort of students, what would you say?
+
+semantic html provides html with structure and meaning making it easy to read and understand whats doing what.
+
+Name two big differences between display: block; and display: inline;.
+
+inline allows for elements to be nested in other elements without braking the flow of content
+while block will take up all space parent allows.
+
+What are the 4 areas of the box model?
+
+content padding border margin. 
+
+While using flexbox, what axis does the following property work on: align-items: center?
+
+parent property
```

Comment:
I looked a little more into this, i think what Lambda is trying to get at is how when you use `justify-content` that affects the layout in the main axis right? and so `align-items` affects the layout in the cross-axis.
Css tricks explains it better than i can, but i hope this helps. 
https://css-tricks.com/almanac/properties/a/align-items/

Reply:
Yea that makes sense. I guess when I read it I interpreted it as what does it need to be placed on for it to work. because if you put it on the child it has zero effect but if on the parent then it works and it affects the child.Thank you for looking more into it though.